Fixes:
 
 client.cc: In member function 'GSList* Gnome::Conf::Client::get_list(const Glib::ustring&, GConfValueType) const':
 client.cc:185: error: cast from 'void*' to 'int' loses precision
 client.cc:188: error: cast from 'void*' to 'gboolean' loses precision

--- gconfmm-2.6.1/gconf/gconfmm/client.cc.orig	2005-06-30 13:10:50.000000000 +0200
+++ gconfmm-2.6.1/gconf/gconfmm/client.cc	2005-06-30 13:09:45.000000000 +0200
@@ -182,10 +182,10 @@
     switch(list_type)
     {
       case GCONF_VALUE_INT:
-        gconf_value_set_int(v,(int)i->data);
+        gconf_value_set_int(v,(int)((long)i->data));
         break;
       case GCONF_VALUE_BOOL:
-        gconf_value_set_bool(v,(gboolean)i->data);
+        gconf_value_set_bool(v,(gboolean)((long)i->data));
         break;
       case GCONF_VALUE_FLOAT:
         gconf_value_set_float(v,*(gdouble*)i->data);
